doc/source/getting-started/basic.rst

```diff
@@ -29,9 +28,8 @@ PROTO files, create coverage reports, and report on system coverage:
 * `example-coverage <https://github.com/pyansys/example-coverage>`_
 * `system-reporting-tool <https://github.com/pyansys/system-reporting-tool>`_
 
-#################
 Quick start guide
-#################
+-----------------
 
 This is a brief overview on how to get started right away with your own PyAnsys
```

Comment:
```suggestion
This is an overview on how to get started right away with your own PyAnsys
```
Overviews are inheritly brief--so brief is redundant.

Reply:
I think it is fine to differentitate between PyAnsys and its organization in GitHub, as this last platform is included inside of the PyAnsys project.